ratelimit/keeper/genesis.go

```diff
@@ -35,17 +39,41 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 		}
 		k.SetPendingSendPacket(ctx, channelId, sequence)
 	}
+
+	// Verify the epoch hour duration is specified
```

Comment:
Shouldn't this be part of the genesis validation (https://github.com/Stride-Labs/ibc-rate-limiting/blob/4e1e00c81edc437c6aa665f31174a44490bbb909/ratelimit/types/genesis.go#L13)?

Reply:
yeah you're right. Updated in latest commit